src/ellswift.rs

```diff
@@ -280,10 +281,15 @@ impl ElligatorSwiftSharedSecret {
     pub const fn as_secret_bytes(&self) -> &[u8; 32] { &self.0 }
 }
 
-/// Represents which party we are in the ECDH, A is the initiator, B is the responder.
-/// This is important because the hash of the shared secret is different depending on which party
-/// we are. In this context, "we" means the party that is using this library, and possesses the
-/// secret key passed to `ElligatorSwift::shared_secret`.
+/// Represents which party we are in the ECDH.
+///
+/// Here `A` is the initiator and `B` is the responder.
```

Comment:
I guess we should've named the enum variants as such. :(

Reply:
https://github.com/rust-bitcoin/rust-secp256k1/issues/741